[c-ares] Build error on x86-windows-static

Package: c-ares:x86-windows-static@1.34.3

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.42.34435.0
-    vcpkg-tool version: 2024-12-09-1005b78fa1bf1dde1a20c2734cba4ea61ca94d9a
    vcpkg-scripts version: 1a66c32c6f 2025-01-02 (13 hours ago)

**To Reproduce**

`vcpkg install c-ares:x86-windows-static`

**Failure logs**

```
-- Using cached c-ares-c-ares-v1.34.3.tar.gz.
-- Cleaning sources at D:/workspace/git/vcpkg/buildtrees/c-ares/src/v1.34.3-4e31af4050.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/workspace/git/vcpkg/downloads/c-ares-c-ares-v1.34.3.tar.gz
-- Applying patch avoid-docs.patch
-- Using source at D:/workspace/git/vcpkg/buildtrees/c-ares/src/v1.34.3-4e31af4050.clean
-- Found external ninja('1.12.1').
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/workspace/git/vcpkg/downloads/tools/cmake-3.30.1-windows/cmake-3.30.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j13
    Working Directory: D:/workspace/git/vcpkg/buildtrees/c-ares/x86-windows-static-dbg
    See logs for more information:
      D:\workspace\git\vcpkg\buildtrees\c-ares\install-x86-windows-static-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/c-ares/portfile.cmake:24 (vcpkg_cmake_install)
  scripts/ports.cmake:196 (include)



```

<details><summary>D:\workspace\git\vcpkg\buildtrees\c-ares\install-x86-windows-static-dbg-out.log</summary>

```
Change Dir: 'D:/workspace/git/vcpkg/buildtrees/c-ares/x86-windows-static-dbg'

Run Build Command(s): "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j13 install
[1/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_android.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_android.c
[2/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_addrinfo2hostent.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_addrinfo2hostent.c
[3/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_getaddrinfo.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_getaddrinfo.c
[4/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_cancel.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_cancel.c
[5/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_free_hostent.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_free_hostent.c
[6/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_freeaddrinfo.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_freeaddrinfo.c
[7/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_conn.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_conn.c
[8/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_destroy.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_destroy.c
[9/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_addrinfo_localhost.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_addrinfo_localhost.c
[10/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_free_string.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_free_string.c
[11/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_close_sockets.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_close_sockets.c
[12/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_data.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_data.c
[13/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_cookie.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_cookie.c
[14/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_getenv.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_getenv.c
[15/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_getnameinfo.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_getnameinfo.c
FAILED: src/lib/CMakeFiles/c-ares.dir/ares_getnameinfo.c.obj 
C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_getnameinfo.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_getnameinfo.c
D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_getnameinfo.c(368): error C4013: “if_indextoname”未定义；假设外部返回 int
D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_getnameinfo.c(368): warning C4047: “==”:“int”与“void *”的间接级别不同
[16/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_gethostbyname.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_gethostbyname.c
[17/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_gethostbyaddr.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_gethostbyaddr.c
[18/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_library_init.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_library_init.c
[19/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_init.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_init.c
[20/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_parse_into_addrinfo.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_parse_into_addrinfo.c
[21/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_hosts_file.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_hosts_file.c
[22/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_qcache.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_qcache.c
[23/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_process.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_process.c
[24/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_options.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_options.c
[25/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_metrics.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_metrics.c
[26/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_query.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_query.c
[27/93] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x86\cl.exe   -DCARES_BUILDING_LIBRARY -DCARES_STATICLIB -DHAVE_CONFIG_H=1 -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -ID:\workspace\git\vcpkg\buildtrees\c-ares\x86-windows-static-dbg -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\include -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib -ID:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\include /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /W4 /w14242 /w14254 /w14263 /w14265 /w14287 /we4289 /w14296 /w14311 /w14545 /w14546 /w14547 /w14549 /w14555 /w14619 /w14640 /w14826 /w14905 /w14906 /w14928 /wd4201 /we4013 /D_DEBUG /MTd /Z7 /Ob0 /Od /RTC1 /showIncludes /Fosrc\lib\CMakeFiles\c-ares.dir\ares_search.c.obj /Fdbin\cares.pdb /FS -c D:\workspace\git\vcpkg\buildtrees\c-ares\src\v1.34.3-4e31af4050.clean\src\lib\ares_search.c
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/c-ares/portfile.cmake
+++ b/ports/c-ares/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO c-ares/c-ares
     REF "v${VERSION}"
-    SHA512 bafe97923a316bb4c33f616448faa04b938f46336cdf26a38ab619faeabcaa88f285e645f4309b928995008e9a0ede232609d58ee344a1729a9e29a9cbe4d94c
+    SHA512 7bd4ca8f1a1b6d7b6662c724315bb5d4ca1d3c19e5ff4e06e3567ea25d5f8fd60f9c5f9ade055f08dc7fc3dec0e40e96f8284207b3e03c0975fd962d4a9fcb47
     HEAD_REF main
     PATCHES
         avoid-docs.patch

--- a/ports/c-ares/portfile.cmake
+++ b/ports/c-ares/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO c-ares/c-ares
     REF "v${VERSION}"
-    SHA512 0c4041ef3ec8e54d11826ae5ea3dbb3752fbd6dcfeb65c3777e56309dc0f4944068c24d54b39cced219c7ee440aae7a6704269558627c26a90001d872d99ded9
+    SHA512 bafe97923a316bb4c33f616448faa04b938f46336cdf26a38ab619faeabcaa88f285e645f4309b928995008e9a0ede232609d58ee344a1729a9e29a9cbe4d94c
     HEAD_REF main
     PATCHES
         avoid-docs.patch

--- a/ports/c-ares/portfile.cmake
+++ b/ports/c-ares/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO c-ares/c-ares
     REF "v${VERSION}"
-    SHA512 87ca8ceda8b4afbc3e68014c6c8604550f17e6b343330801c396aae393f1c98747e65d3459db7b9626abc73f1216422e794a0d0f044dd1fda3efda4005b0ef1f
+    SHA512 5924d706341e35ad467bf48c88a8f53617d5fb94546b1d9dd24a7091cc84763e52f9bbdfa39abdf1aa55099d4b87116af56bf584b99a5c8f5e554ecf9d7bc8a8
     HEAD_REF main
     PATCHES
         avoid-docs.patch

--- a/ports/c-ares/vcpkg.json
+++ b/ports/c-ares/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "c-ares",
-  "version-semver": "1.34.3",
+  "version-semver": "1.34.4",
   "description": "A C library for asynchronous DNS requests",
   "homepage": "https://github.com/c-ares/c-ares",
   "license": "MIT-CMU",

--- a/ports/c-ares/vcpkg.json
+++ b/ports/c-ares/vcpkg.json
@@ -1,11 +1,9 @@
 {
   "name": "c-ares",
-  "version-semver": "1.31.0",
-  "port-version": 1,
+  "version-semver": "1.33.0",
   "description": "A C library for asynchronous DNS requests",
   "homepage": "https://github.com/c-ares/c-ares",
   "license": "MIT-CMU",
-  "supports": "!uwp",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/ports/c-ares/vcpkg.json
+++ b/ports/c-ares/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "c-ares",
-  "version-semver": "1.34.2",
+  "version-semver": "1.34.3",
   "description": "A C library for asynchronous DNS requests",
   "homepage": "https://github.com/c-ares/c-ares",
   "license": "MIT-CMU",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1433,8 +1433,8 @@
       "port-version": 5
     },
     "c-ares": {
-      "baseline": "1.31.0",
-      "port-version": 1
+      "baseline": "1.33.0",
+      "port-version": 0
     },
     "c-dbg-macro": {
       "baseline": "2020-02-29",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1457,7 +1457,7 @@
       "port-version": 6
     },
     "c-ares": {
-      "baseline": "1.34.3",
+      "baseline": "1.34.4",
       "port-version": 0
     },
     "c-dbg-macro": {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1457,7 +1457,7 @@
       "port-version": 6
     },
     "c-ares": {
-      "baseline": "1.34.2",
+      "baseline": "1.34.3",
       "port-version": 0
     },
     "c-dbg-macro": {

--- a/versions/c-/c-ares.json
+++ b/versions/c-/c-ares.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "e7450d43c71f90bacb38c23f1172ac6c9cccb083",
+      "version-semver": "1.34.4",
+      "port-version": 0
+    },
+    {
       "git-tree": "c01c2a71a29047cc895afe3286f691c68f36927b",
       "version-semver": "1.34.3",
       "port-version": 0

--- a/versions/c-/c-ares.json
+++ b/versions/c-/c-ares.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "c01c2a71a29047cc895afe3286f691c68f36927b",
+      "version-semver": "1.34.3",
+      "port-version": 0
+    },
+    {
       "git-tree": "78f3eb04280f992a8af0ebc68f67d965542ebffc",
       "version-semver": "1.34.2",
       "port-version": 0

--- a/versions/c-/c-ares.json
+++ b/versions/c-/c-ares.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "05a0806e25d8f4ab5a0e147c157bfffb345b4da8",
+      "version-semver": "1.33.0",
+      "port-version": 0
+    },
+    {
       "git-tree": "6c07bb206a33f3005265a3e2db4c697a3097f8f3",
       "version-semver": "1.31.0",
       "port-version": 1

